packages/components/src/drop-zone/style.scss

```diff
@@ -61,3 +61,11 @@
 .components-drop-zone__provider {
 	height: 100%;
 }
+
+[data-is-dragging] * {
+	pointer-events: none;
+}
+
+[data-is-drop-target] {
+	pointer-events: all;
+}
```

Comment:
Any concern about these being too global?  should we have a `wp` prefix there or something?

Reply:
I'm fine  with these but the issue is that they don't target any particular id or class. (same issues applies to is-something). In other words, I'd prefer `.components-dropzone[data-is-drop-target]` but I guess we Don't always have an element on which to apply a class?

Not a bit concern by my main thinking is that these names seem generic enough that they could be used by other libraries...

Reply:
Yeah. Tbh, I'm hoping to avoid these attributes entirely in a follow up. I can prefix them for now.

Reply:
I prefixed it